builder/azure/common/env.go

```diff
@@ -0,0 +1,14 @@
+package common
+
+import "os"
+
+const DebugLogsEnvVar string = "PACKER_AZURE_DEBUG_LOGS"
```

Comment:
nit: Prefix with Azure to denote these are Azure debug log and drop the `s` for consistency with _LOG
```suggestion
const AzureDebugLogsEnvVar string = "PACKER_AZURE_DEBUG_LOG"
```